src/com/sun/ts/lib/harness/keyword.properties

```diff
@@ -242,6 +256,7 @@ com/sun/ts/tests/servlet/ee/platform/deploy/ejbref/single = javaee_optional java
 
 com/sun/ts/tests/webservices12 = webservices javaee
 com/sun/ts/tests/webservices12/ejb/annotations/WSEjbWebServiceProviderTest= webservices javaee_optional xml_binding
+com/sun/ts/tests/webservices12/ejb/descriptors/WSEjbOverrideWSRefWithDDsTest = rmi_iiop
```

Comment:
Which class does this test use for rmi_iiop?  I could be missing something but I only see java.rmi.Remote which can also be used for EJB.

```
grep -ri --no-filename import | sort | uniq
import com.sun.javatest.Status;
import com.sun.ts.lib.harness.*;
import com.sun.ts.lib.porting.*;
import com.sun.ts.lib.util.*;
import com.sun.ts.tests.jaxws.common.*;
import jakarta.ejb.CreateException;
import jakarta.ejb.EJBHome;
import jakarta.ejb.EJBObject;
import jakarta.ejb.SessionBean;
import jakarta.ejb.SessionContext;
import jakarta.ejb.Stateless;
import jakarta.jws.*;
import jakarta.jws.WebService;
import jakarta.xml.ws.Service;
import jakarta.xml.ws.WebServiceException;
import jakarta.xml.ws.WebServiceRef;
import java.rmi.Remote;
import java.rmi.RemoteException;
import java.util.Iterator;
import java.util.Properties;
import javax.naming.InitialContext;
import javax.xml.namespace.QName;
```

Reply:
done